.circleci/config.yml

```diff
@@ -701,7 +701,7 @@ jobs:
         # debug: shows how many time each test was executed
       - run: jq -s ".[]|.testenvs|keys|.[]" /tmp/workspace/* | grep -v GLOB | sed 's/"//g' | sort | uniq -c | sort -rn
         # list all executed test
-      - run: jq -s ".[]|.testenvs|keys|.[]" /tmp/workspace/* | grep -v GLOB | sed 's/"//g' | sort | uniq | tee all_executed_tests
+      - run: jq -s ".[]|.testenvs|keys|.[]" /tmp/workspace/* | grep -v GLOB | sed 's/"//g' | sort | uniq | grep -v '^\.' | tee all_executed_tests
```

Comment:
why is this changing?

did we change it before because of an update in `virtualenv` and now rolling it back?

Reply:
This is because the `requires` parameter creates a new "hidden" pre-step that is listed here.
This is a special one and it starts with a `.` so it's an easy way to identify.
The same happens when you use `isolated_build` (see https://github.com/DataDog/dd-trace-py/pull/1203)